### PR TITLE
M109 Florida Onboarding

### PR DESCRIPTION
**Pull request by mauricereeves**
This PR does creates the schema for #109. Also introduces a few changes for how documentation is created.

## Changes
- Adds the Florida schema
- Improves the documentation generation 

## Checklist
- [x] Tested locally?
- [ ] Added new dependencies?
- [x] Updated the docs?
- [ ] Added a test?
- [x] Did you check for sensitive data, and remove any?
- [ ] Are additional approvals needed for this change?
- [ ] Are there potential vulnerabilities or licensing issues with any new dependencies introduced?

### CODE REVIEW

File: prime-router/metadata/schemas/FL/fl-covid-19.schema [hunk 1]
```diff
@@ -0,0 +1,368 @@
...
+            - N
+            - U
+
+    # column BJ
+    # nonstandard
+    - name: name_of_school
+      type: TEXT
+      csvFields: [{ name: "What is the name of the school" }]
+      documentation: AOE question for Florida.
+
```
Comment (RickHawesUSDS) [inline prime-router/metadata/schemas/FL/fl-covid-19.schema:367]: Worth having a discussion with SimpleReport. They are defining `ordering_facility_name` as the place this stuff should be kept. Should this be the place for this name?

File: prime-router/metadata/schemas/FL/fl-covid-19.schema [hunk 2]
```diff
@@ -0,0 +1,368 @@
...
+      type: TEXT
+      csvFields: [{ name: "Is the patient a student, teacher, or other faculty member" }]
+      documentation: |
+        AOE question for Florida. Expects one of the following values:
+            - Student
+            - Teacher
+            - Other (Faculty Member)
+            - N
+            - U
+
```
Comment (RickHawesUSDS) [inline prime-router/metadata/schemas/FL/fl-covid-19.schema:360]: Could define a `CODE` with AltValues here.

**Review by RickHawesUSDS — CHANGES_REQUESTED** [review]
A couple of suggestions.

File: prime-router/metadata/schemas/FL/fl-covid-19.schema (hunk 1 above)
Comment (mauricereeves) [reply prime-router/metadata/schemas/FL/fl-covid-19.schema:328]: I think this is resolved. We can close out this point.

File: prime-router/metadata/schemas/FL/fl-covid-19.schema (hunk 2 above)
Comment (mauricereeves) [reply prime-router/metadata/schemas/FL/fl-covid-19.schema:321]: I will convert these to CODE and add AltValues

**Review by MauriceReeves-usds — APPROVED** [review]

**Review by RickHawesUSDS — APPROVED** [review]

**Pull request closed without merging**